### DISCUSSION TITLE
safari webview crashes when displaying 3d models

### DISCUSSION THREAD

**Discussion by emrhnksck** [google/model-viewer#3925] [Q&A]
### Description

I have problem when render the 3d models with model-viewer. If i embed the textures of model in glb files, create the model and hit the model detail for display 3d model, safari webview crashes and shows just full white screen. I'm not facing this problem on android or web it happens in just ios. Maybe its memory issue or model-viewer doesn't support display models created with embedded texture files? 

### Version
![IMG_1231](https://user-images.githubusercontent.com/58311722/199920139-01a3428f-8d8b-4cfe-8e28-e54fc91bb107.jpg)

- model-viewer: v2.0.2

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook
![IMG_1231](https://user-images.githubusercontent.com/58311722/199920307-4dc088ed-5cea-4cdc-a42d-af2aa50117e5.jpg)

**Comment by elalish** [comment]
Nice screenshots! I'd guess a memory problem? Have you tried with a small/simplified model? Any console logs?
